app/src/androidTest/java/com/android/streetworkapp/ui/profile/ProfileComponentsTest.kt

```diff
@@ -36,6 +66,14 @@ class ProfileFriendsListTest {
         .assertExists()
         .assertIsDisplayed()
         .assertTextEquals("Score: ${friend.score}")
+
+    composeTestRule
+        .onNodeWithTag("friendStatus")
+        .assertExists()
+        .assertIsDisplayed()
+        .assertTextEquals("Definitely not a bot")
```

Comment:
Improvement: Given that not everyone has the figma nearby, I think that hardcoding funny values without specifying what they are will confuse some teammates

Reply:
I made default values explicit using constants (upper case vals), is that good for you ?